org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -669,22 +939,60 @@ package lang {
 	 */	
 	class Number {
 	
+		/** 
+		 * Returns the greater number between two
+		 * Example:
+		 * 		5.max(8)    ==> returns 8 
+		 */
 		method max(other) = if (self >= other) self else other
+		
+		/** Returns the lower number between two. @see max */
 		method min(other) = if (self <= other) self else other
 		
+		/**
+		 * Given self and a range of integer values, returns self if it is in that range
+		 * or nearest value from self to that range 
+		 *
+		 * Examples
+		 * 4.limitBetween(2, 10)   ==> returns 4, because 4 is in the range
+		 * 4.limitBetween(6, 10)   ==> returns 6, because 4 is not in range 6..10, and 6 is nearest value to 4
+		 * 4.limitBetween(1, 2)    ==> returns 2, because 4 is not in range 1..2, but 2 is nearest value to 4
+		 *
+		 */   
 		method limitBetween(limitA,limitB) = if(limitA <= limitB) 
 												limitA.max(self).min(limitB) 
 											 else 
-
 											 	limitB.max(self).min(limitA)
 
+		/** @private */
 		override method simplifiedToSmartString(){ return self.stringValue() }
+		
+		/** @private */
 		override method internalToSmartString(alreadyShown) { return self.stringValue() }
+		
+		/** Returns true if self is between min and max */
 		method between(min, max) { return (self >= min) && (self <= max) }
```

Comment:
Jajajaj (or should I laugh also Hahahaha!!!)
Answers in Smalltalk / Returns is Java
"I represent" in Smalltalk / "this object" in Java
Ok with expression-oriented thinking. And for the rest I'll let you change whenever you want.